server/services/approverCaseloadService.ts

```diff
@@ -107,77 +75,91 @@ export default class ApproverCaseloadService {
         }
       })
       .filter(offender => offender.nomisRecord)
-  }
+      .map(offender => {
+        return this.toApprovalCase(offender, licences)
+      })
 
-  private async mapResponsibleComsToCases(caseload: ManagedCaseForApproval[]): Promise<ApprovalCase[]> {
     const comUsernames = caseload
-      .map(
-        offender =>
-          offender.licences.find(l => offender.licences.length === 1 || l.status !== LicenceStatus.ACTIVE).comUsername
-      )
+      .map(({ comUsernameOnLicence }) => comUsernameOnLicence)
       .filter(comUsername => comUsername)
 
     const coms = await this.communityService.getStaffDetailsByUsernameList(comUsernames)
 
     return caseload
       .map(offender => {
-        const responsibleCom = coms.find(
```

Comment:
extracted this out into a helper function that solely gets the PP for a licence to enrich which is a bit nicer